src/lwc/accordionSection/accordionSection.js-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="accordionSection">
+    <apiVersion>46.0</apiVersion>
```

Comment:
The api version needs to 49 or 50 (all files in this project will need to be the same api version before they can be merged into master.).